### DISCUSSION TITLE
TV

### DISCUSSION THREAD

**Discussion by jdiegmueller** [punxaphil/custom-sonos-card#65] [Q&A] [answered]
Is there any way to use the custom-sonos-card to change input to "TV"?  I have a Sonos Arc + Sub Gen3 + 2xPlay 5s, and often need to be able to change to the TV input.  I can do this from a media_player interface, but can't find a way to do this with custom-sonos-card.  Thanks!

Example:
![image](https://user-images.githubusercontent.com/18660571/165497213-f2f33910-c053-46fe-880a-79e26a88d944.png)

**Comment by punxaphil** [comment, marked as answer]
https://github.com/johanfrick/custom-sonos-card#usage
```
customSources: # Main usecase is probably to set tv media player to play TV sound
  media_player.tv: # set this to 'all' to show the custom source for all players
    - title: TV
      thumbnail: https://cdn-icons-png.flaticon.com/512/716/716429.png
```

**Reply by jdiegmueller** [reply]
Thank you.  A big DUH that I didn't find that.

**Reply by punxaphil** [reply]
No worries
